.github/workflows/pre-commit.yml

```diff
@@ -1,9 +1,7 @@
 name: pre-commit
 
 on:
-  pull_request:
   push:
-    branches: [main]
```

Comment:
these changes seem to make no CI at all run for prs submitted from forks: https://github.com/pymc-devs/pymc/pull/5617